IdentityCore/src/requests/MSIDInteractiveTokenRequest.m

```diff
@@ -77,6 +80,7 @@ - (nullable instancetype)initWithRequestParameters:(nonnull MSIDInteractiveToken
         _tokenCache = tokenCache;
         _accountMetadataCache = accountMetadataCache;
         _tokenResponseHandler = [MSIDTokenResponseHandler new];
+        _lastRequestTelemetry = [MSIDLastRequestTelemetry sharedInstance];
```

Comment:
just curious, how would current telemetry get passed to authorize request? I'm seeing we track result of interactive request.

Reply:
We will send current request to /authorize endpoint in separate pr.